Carnival of Awussu implementation

Arveanor opened the pull request.

Arveanor committed:
Merge branch 'master' into implement-awussu-carnival

Arveanor committed:
Working changes

Still need to finish pretty lad and implement pretty lass, otherwise hooked into summer fair events

Arveanor committed:
Finish up adding Carnival of Awussu

Finished up the new events and added the pictures

faiuwle commented on AfricanReligions/decisions/afr_decisions.txt:
```diff
@@ -242,7 +242,7 @@ decisions = {
     become_wangara = {
         only_playable = yes
-		is_high_prio = yes
+		is_high_prio = years
```
???

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+	only_playable = yes
+
+	trigger = {
+		religion = amazigh
```
Or `amazigh_reformed`, yes?

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+
+	trigger = {
+		religion = amazigh
+		OR = {
```
A simpler way to do this is to use `prefers_men_trigger = yes` (and `prefers_men_trigger = no` for the reverse) - it's more likely to be compatible with mods that change how sexuality works.

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+			NOT = { has_character_flag = fair_event_happened }
+			had_character_flag = {
+				flag = fair_event_happened
+				days = 3hjk
```
???

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+	}
+
+	immediate = {
+		set_character_flag = fair_event_AFRREL035
```
Can this flag be more readable, like, say, `fair_event_pretty_lad` or something?  It will make it easier to identify in-game.  I know the vanilla ones aren't great either.

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+			if = {
+				limit = { ROOT = { trait = homosexual } }
+				add_trait = homosexual
+			}
```
We probably want another clause here removing `homosexual` in the other case, if they happen to be generated with (not sure if that can happen, but it doesn't hurt to check.
```suggestion
			}
			else = { remove_trait = homosexual }
```

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -65,6 +65,20 @@ EVTOPTB_AFRREL029;These Carthaginians, when will they learn?;;;;;;;;;;;;;x
+EVTDESC_AFRREL037;In the old Amazigh calendar, the great heat wave that hits the lands of the Maghreb when the month of Ghusht arrives is known as Awussu. The occasion is celebrated with a great carnival, and this year I have decided to sponsor the entire celebration on my own, to the joy of the commoners.\n\nI give the order to light the big bonfires for my capital and give gifts to the people w ...
+EVTOPTA_AFRREL037;We will have so much fun!;;;;;;;;;;;;;x
+EVTTITLE_AFRREL038;Carnival Ends;;;;;;;;;;;;;x
+EVTDESC_AFRREL038;As the heat wave ends and the people calm down, another carnival comes to an end. This is an old celebration, going back to the days of old Carthage and with good reason. The fun it�s always good for the spirit.;;;;;;;;;;;;;x
```
it's -> is

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -65,6 +65,20 @@ EVTOPTB_AFRREL029;These Carthaginians, when will they learn?;;;;;;;;;;;;;x
+EVTOPTA_AFRREL038;Until next year!;;;;;;;;;;;;;x
+EVTTITLE_AFRREL035;Carnival, pretty lad;;;;;;;;;;;;;x
+EVTDESC_AFRREL035;As it is tradition to do in carnival times, I decide to take a plunge in the water for good luck. It is then that I see that I am not alone. His muscles seem to be sculpted in marble and his smile could make any heart melt. They say that the waters in Awussu are good for fertility, and he seems more than willing to put his lips in the royal chalice.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL035;Well, if the gods sent him to me ...;;;;;;;;;;;;;x
```
Space between "me" and the elipsis

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,226 @@
+#Carnival of Awussu, pretty lad
+character_event = {
+	id = AFRREL.035
+	title = EVTTITLE_AFRREL035
```
Let's not use titles for these events.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -65,6 +65,20 @@ EVTOPTB_AFRREL029;These Carthaginians, when will they learn?;;;;;;;;;;;;;x
+EVTOPTA_AFRREL035;Well, if the gods sent him to me ...;;;;;;;;;;;;;x
+EVTOPTB_AFRREL035;Get away from me! I am a decent [Root.GetManWoman];;;;;;;;;;;;;x
+EVTTITLE_AFRREL036;Carnival, pretty lass;;;;;;;;;;;;;x
+EVTDESC_AFRREL036;As it is tradition to do in carnival times, I decide to take a plunge in the water for good luck. It is then that I see that I am not alone. Her curvaceous body steals my breath and her breasts awaken all my desires. They say that the waters in Awussu are good for fertility, and she seems more than willing to play with the royal scepter.;;;;;;;;;;;;;x
```
The sexual innuendo seems to be baked into these based on the assumption that ROOT is straight.  What if we factored out "drink from the royal chalice" and "play with the royal scepter" into a customizable localization that picked based on gender and call it as e.g. `[Root.GetAppropriateSexualInnuendo]` or whatever?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -65,6 +65,20 @@ EVTOPTB_AFRREL029;These Carthaginians, when will they learn?;;;;;;;;;;;;;x
+EVTOPTB_AFRREL035;Get away from me! I am a decent [Root.GetManWoman];;;;;;;;;;;;;x
+EVTTITLE_AFRREL036;Carnival, pretty lass;;;;;;;;;;;;;x
+EVTDESC_AFRREL036;As it is tradition to do in carnival times, I decide to take a plunge in the water for good luck. It is then that I see that I am not alone. Her curvaceous body steals my breath and her breasts awaken all my desires. They say that the waters in Awussu are good for fertility, and she seems more than willing to play with the royal scepter.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL036;Well, if the gods sent her to me ...;;;;;;;;;;;;;x
```
Same elipsis problem as above

faiuwle commented on AfricanReligions/localisation/afr_general_localizations.csv:
```diff
@@ -203,6 +203,8 @@ vassalize_inna;Vassalize the Inna;;;;;;;;;;;;;x
 exalt_inna;Exalt the Inna;;;;;;;;;;;;;x
 exalt_inna_desc;For too long our spiritual mother, the Inna, has seen her power limited by secular rulers. We must grant her full religious sovereignty, which will bring prosperity and glory to our realms.;;;;;;;;;;;;;x
+hold_carnival_of_awussu;Hold the Carnival of Awussu;;;;;;;;;;;;;x
+hold_carnival_of_awussu;During Awussu's heat wave the festival of the same name is celebrated. It is a tradition that dates back to the Phoenician times.;;;;;;;;;;;;;x
```
You meant `hold_carnival_of_awussu_desc`.

Arveanor committed:
Remove old afr events files

These were never intended to be re-added, just cleaning up.

faiuwle reviewed (changes requested):
Some brief comments on this version.  We still need the relocalizations, and I haven't looked at it in-game yet to check the event texts.

Arveanor committed:
Merge branch 'master' into implement-awussu-carnival

afr_decisions.txt just needed two unrelated decisions added at the same line to be resolved

Arveanor committed:
add summer fair localisations for awussu

Arveanor committed:
Add cropped image version

Arveanor committed:
Fix issues on PR

Arveanor committed:
Localisation and picture changes

Lover events use custom localisation based on gender and event images are updated.

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,214 @@
+    id = AFRREL.037
+    title = EVTTITLE_AFRREL037
+    desc = EVTDESC_AFRREL037
+    picture = GFX_evt_afr_awussu_carnival
```
This picture is too big.  It needs to be 150x450.

faiuwle commented on AfricanReligions/events/afr_awussu_events.txt:
```diff
@@ -0,0 +1,214 @@
+	}
+
+	immediate = {
+		set_character_flag = fair_event_lover_lass
```
These flags are swapped on these two events.

faiuwle commented on AfricanReligions/localisation/000_vanilla_overrides.csv:
```diff
@@ -22,4 +22,6 @@ pick_main_deity;[PickPatronDeityReligionCheck];;;;;;;;;;;;x
-patron_deity_bori;patron bori to follow;;;;;;;;;;;;x
\ No newline at end of file
+patron_deity_bori;patron bori to follow;;;;;;;;;;;;x
+awussu_female_innuendo;drink from the royal chalice.
```
These need semicolons at the ends.

faiuwle reviewed (changes requested):
A few notes.  Also, please fix the merge conflict.

Arveanor committed:
Fix PR issues

resized image, added localisation line endings, fixed flag names

Arveanor committed:
Merge branch 'master' into implement-awussu-carnival

faiuwle committed:
Merged master

faiuwle committed:
Merged master
